odoo/addons/base/res/res_currency.py

```diff
@@ -28,6 +35,8 @@ class Currency(models.Model):
     position = fields.Selection([('after', 'After Amount'), ('before', 'Before Amount')], default='after',
         string='Symbol Position', help="Determines where the currency symbol should be placed after or before the amount.")
     date = fields.Date(compute='_compute_date')
+    currency_name = fields.Char(string="Currency", help="Currency Name")
+    sub_currency_name = fields.Char(string="Sub Currency", help="Sub Currency Name")
 
```

Comment:
rename fields into:
currency_unit_label and currency_subunit_label

Reply:
Done. Thanks